AlgebraInLean/Chapter01/Sheet06.lean

```diff
@@ -0,0 +1,89 @@
+import AlgebraInLean.Chapter01.Sheet05
+
+namespace AlgebraInLean
+
+/-
+The next example of groups we will go over are the dihedral groups Dₙ. Similarly to the cyclic
+groups, the dihedral groups also represent symmetries on a regular n-gon. However, Dₙ includes both
+rotations and reflections.
+-/
+
+/--
+The dihedral group Dₙ has 2n elements: the n rotations already in Cₙ, and the reflections of all
+of these rotations (reflection first, then rotation).
+-/
+inductive Dihedral (n : ℕ) [NeZero n] : Type
```

Comment:
Could you explain this definition in more detail? It may look fairly unintuitive for someone who isn't used to working with inductive types